src/Poster-Container.js

```diff
@@ -9,7 +9,9 @@ const PosterContainer = ({movies, showMovieSummary}) => {
 
       <Poster
         imagePath={movie.poster_path}
+        backdropPath={movie.backdrop_path}
         title={movie.title}
+        rating={Math.round(movie.average_rating)}
```

Comment:
Maybe we can see what toFixed() looks like here. This looks good but we might want to keep keep the first decimal place.